tests/modules/programs/pls/bash.nix

```
@@ -0,0 +1,28 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    programs = {
+      bash.enable = true;
+
+      pls = {
+        enable = true;
+        enableAliases = true;
+      };
+    };
+
+    test.stubs.pls = { };
+
+    nmt.script = ''
+      assertFileExists home-files/.bashrc
+      assertFileContains \
+        home-files/.bashrc \
+        "alias ls='${pkgs.pls}/bin/pls'"
+      assertFileContains \
+        home-files/.bashrc \
+        "alias ll='${pkgs.pls}/bin/pls -d perms -d user -d group -d size -d mtime -d git'"
```

Using the stubbed package,
```suggestion
        "alias ls='@pls@/bin/pls'"
      assertFileContains \
        home-files/.bashrc \
        "alias ll='@pls@/bin/pls -d perms -d user -d group -d size -d mtime -d git'"
```